coderd/database/dbmem/dbmem.go

```diff
@@ -6633,6 +6633,15 @@ func (q *FakeQuerier) InsertProvisionerJobLogs(_ context.Context, arg database.I
 	return logs, nil
 }
 
+func (*FakeQuerier) InsertProvisionerJobTimings(_ context.Context, arg database.InsertProvisionerJobTimingsParams) ([]database.ProvisionerJobTiming, error) {
+	err := validateDatabaseType(arg)
+	if err != nil {
+		return nil, err
+	}
+
+	return nil, nil
```

Comment:
Didn't really seem necessary since I can't currently test the full roundtrip including calls to the db.
Happy to add it though if you want it for completeness' sake.